### ISSUE TITLE
vim9script: 'silent' before 'execute' is ignored

### ISSUE THREAD

**Issue by bfrg** [vim/vim#6530]
**Describe the bug**
A `silent` before `execute` is ignored in `vim9script`. This is different from legacy Vim script. If this is intentional it should probably be mentioned somewhere in the documentation.

**To Reproduce**
1. Run `vim --clean -S test.vim` with
   ```vim
   vim9script
   def Test()
       silent execute 'echomsg "test"'
   enddef
   Test()
   ```
2. Result: The message is added to the message history regardless of `silent`.

---

It works when `silent` appears after `execute`:
```vim
def Test()
    execute 'silent echomsg "test"'
enddef
```

**Expected behavior**
I expected the same behavior as in legacy Vim script.

**Environment**
Vim version `8.2.1287`

**Comment by brammool** [comment]
command modifiers are not implemented yet.

**Comment by lacygoill** [comment]
Possibly fixed by [8.2.1894](https://github.com/vim/vim/releases/tag/v8.2.1894).

**Comment by brammool** [comment]
Yes, the example works properly now
